src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccountDeployment.cs

```diff
@@ -61,6 +61,15 @@ public class NewAzureCognitiveServicesAccountDeploymentCommand : CognitiveServic
             Mandatory = true,
             ParameterSetName = DefaultParameterSet,
             ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Cognitive Services Deployment Sku.")]
+        [ValidateNotNullOrEmpty]
+        public Sku Sku { get; set; }
+
+        [Parameter(
+            Position = 4,
```

Comment:
Since this cmdlet introduced a new property `Sku`, the help doc should also be re-generated via platyPS due to syntax change.